test/modules/finance.spec.ts

```diff
@@ -338,47 +343,31 @@ describe('finance', () => {
           expect(bitcoinAddress).toSatisfy(isBtcAddress);
         });
 
-        it.each`
-          type                          | expectedPrefix
-          ${BitcoinAddressType.Legacy}  | ${'1'}
-          ${BitcoinAddressType.Segwit}  | ${'3'}
-          ${BitcoinAddressType.Bech32}  | ${'bc1'}
-          ${BitcoinAddressType.Taproot} | ${'bc1p'}
-        `(
+        it.each([
+          [BitcoinAddressType.Legacy, legacy],
+          [BitcoinAddressType.Segwit, segwit],
+          [BitcoinAddressType.Bech32, bech32],
+          [BitcoinAddressType.Taproot, taproot],
+        ])(
           'should handle the type = $type argument',
-          ({
-            type,
-            expectedPrefix,
-          }: {
-            type: BitcoinAddressType;
-            expectedPrefix: string;
-          }) => {
+          (type: BitcoinAddressType, regex: RegExp) => {
             const bitcoinAddress = faker.finance.bitcoinAddress({
               type,
             });
 
             expect(bitcoinAddress).toBeTruthy();
             expect(bitcoinAddress).toBeTypeOf('string');
             expect(bitcoinAddress).toSatisfy(isBtcAddress);
-            expect(bitcoinAddress).toSatisfy((v: string) =>
-              v.startsWith(expectedPrefix)
-            );
+            expect(bitcoinAddress).toSatisfy((v: string) => regex.test(v));
```

Comment:
I suggest using `toMatch` when working with RegExp as the potential error message gives more detailed information than `toSatisfy` in this case.

```suggestion
            expect(bitcoinAddress).toMatch(regex);
```

Reply:
All tests changed to use regexes and `toMatch` where possible [84c5b0e](https://github.com/faker-js/faker/pull/2922/commits/84c5b0ea714f79459ed14efd40bef89a327cab1a)